Add AppGameKit (#239)
SteamDatabase/FileDetectionRuleSets: descriptions/Engine.AppGameKit.md

> [**AppGameKit**](https://www.appgamekit.com/) offers a high level coding and programming tool which aims to be easy for beginners to learn.

Reviewer: i'm not a fan of that sentence, that suggestion isn't perfect either 
```suggestion
[**AppGameKit**](https://www.appgamekit.com/) is a game engine that offers a high level coding and programming tool which aims to be easy for beginners to learn.
```

> [**AppGameKit**](https://www.appgamekit.com/) is a game engine that offers a high level coding and programming tool which aims to be easy for beginners to learn.

Reviewer: this will be even better
```suggestion
[**AppGameKit**](https://www.appgamekit.com/) is a game engine developed by "The Game Creators" which aims to be easy for beginners to learn.
```
